app/models/concerns/lets_encrypt/certificate_issuable.rb

```diff
@@ -26,7 +26,6 @@ def csr
 
     def create_certificate
       order.finalize(csr: csr)
-      sleep 1 while order.status == 'processing'
```

Comment:
Need to reload the order or wont work.
      while order.status == 'processing'
        sleep 1
        order.reload
      end

Reply:
According to LetsEncrypt Ruby client...